cmd/agent/dist/conf.d/snmp.d/auto_conf.yaml

```diff
@@ -114,6 +114,11 @@ instances:
     #
     collect_device_metadata: "%%extra_collect_device_metadata%%"
 
+    ## @param collect_topology - boolean - optional - default: false
+    ## Enabled collection of topology data.
+    #
+    collect_topology: "%%extra_collect_topology%%"
```

Comment:
Do we need to advertise it in the auto_conf file?

Reply:
@FlorianVeaux We need at least to have `collect_topology: "%%extra_collect_topology%%"`, otherwise the config won't be forwarded from snmp_listener to snmp integration, the comment is optional though.